src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -731,7 +721,10 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
 
 exit:
     // TODO: Add MapBLEError to return the correct error code
```

Comment:
Is this TODO still necessary? If so, can we apply the change?

Reply:
this TODO might still be necessary, having said that let's put it across in another PR since this PR is focused on the sleep issue.

Reply:
Can you open an task under this story that we need to fix this?
https://jira.silabs.com/browse/MATTER-4426

Reply:
https://jira.silabs.com/browse/MATTER-4426